src/Microsoft.AspNetCore.Mvc.Abstractions/ModelBinding/ModelMetadata.cs

```diff
@@ -296,6 +297,13 @@ public string PropertyName
         public abstract string TemplateHint { get; }
 
         /// <summary>
+        /// Gets an <see cref="IShouldValidate"/> implementation that indicates whether this model should be validated.
+        /// If <c>null</c>, properties with this <see cref="ModelMetadata"/> are validated.
+        /// </summary>
+        /// <value>Defaults to <c>null</c>.</value>
+        public abstract IShouldValidate ShouldValidate { get; }
```

Comment:
Decided offline that only the name needs to change.

Going with `IPropertyValidationFilter`. If this was a(nother) MVC do-over, might consider a new name for `IPropertyFilterProvider` (something more obviously binding-related) but not touching that now.